<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatBox</class>
 <widget class="QWidget" name="chatBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="chatLabel">
       <property name="text">
        <string>Enter Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="userChat">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="chat">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="listOnline">
     <property name="text">
      <string>Online</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="fileSendButton">
     <property name="text">
      <string>FileSend</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>chatBox</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>userChat</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSendButton</sender>
   <signal>clicked()</signal>
   <receiver>userChat</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
